docs/pages/machine-id/platform-guides/aws.mdx

```diff
@@ -3,4 +3,51 @@ title: Deploying Machine ID on AWS
 description: How to install and configure Machine ID on an AWS EC2 instance
 ---
 
-TODO
\ No newline at end of file
+This page contains relevant background information and specific steps for
+deploying Machine ID on Amazon Web Services.
+
+## Background
+
+### The `iam` join method
+
+TODO: Brief overview of how the join method works for this platform.
+
+### Elastic Kubernetes Service
+
+Whilst the guide on this page focuses explicitly on deploying Machine ID on an
+EC2 instance, it is also possible to use the `iam` join method with
+workloads (e.g Pods) running on an EKS Kubernetes cluster.
+
+It is a requirement that
+[IAM Roles for Service Accounts (IRSA)](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html)
+has been configured for the cluster and the Kubernetes service account that will
+be used by the `tbot` pod.
+
+See the [Kubernetes platform guide](kubernetes.mdx) for further guidance
+on deploying Machine ID as a workload on Kubernetes.
```

Comment:
> If I have chosen this page, that would mean that I have seen the "Virtual Machines" H2 of the menu page and clicked for instructions to "install and configure Machine ID on an AWS EC2 instance".

Ah, interesting you raise this because I'd semi-forgotten that I'd made this more "explicit" in referring to an EC2 instance on the higher pages. That definitely makes this less relevant.

> What would you think of a separate platform guide for using the iam join method on Kubernetes? 

At this time, I don't think I have the resources to handle this since this would be another three guides to maintain - even if it's more verbose.